atomata/capstone_WebClient
# Updated workbench and tests

### Summary 
Removed the description field in exchange for a date modified field

### Details
- Removed description field
- Added date modified field
- Added new tests
### How to test?
1. Log in and go to the Workbench page
2. Create an experience
3. Save the experience
4. Return to Workbench page and check if the Date Modified field is correct
5. Edit that experience and save again
6. Return to Workbench page and check if the Date Modified field has been updated and is still correct
7. Also run some tests
### Checks
